components/UI/ServiceLink.js

```diff
@@ -4,19 +4,21 @@ import Link from 'next/link';
 import styled from 'styled-components';
 
 import LockIcon from '../../assets/icons/lock.svg';
-import { ServiceTable } from '../../components/UI/ServiceTable';
+import { ServiceTable } from './ServiceTable';
 
 const LockIconResized = styled(LockIcon)`
   display: block;
   transform: scale(0.9);
 `;
 
-const EtherpadListEntry = forwardRef(({ roomId, padName, passwordProtected, selected }, ref) => {
-    const { t } = useTranslation('write');
+const ServiceLink = forwardRef(({ roomId, name, selected, path, passwordProtected }, ref) => {
+    const { t } = useTranslation();
 
     return (
         <ServiceTable.Row>
-            <ServiceTable.Cell selected={selected}><Link ref={ref} disabled href={`/write/${roomId}`}>{ padName }</Link></ServiceTable.Cell>
+            <ServiceTable.Cell selected={selected}>
+                <Link ref={ref} disabled href={`/${path}/${roomId}`}>{ name }</Link>
```

Comment:
Let's generalize this a bit more and only pass `path` (and **not** `roomId` anymore) as a parameter to `ServiceLink`.
`ServiceLink` should not automatically assume that it's always `/<x>/<roomId>` -- instead the consuming page or component will take care of that logic itself by passing `path={'/etherpad/${roomId}'}` as a parameter instead.